src/Control/Monad/Morph.hs

```diff
@@ -116,6 +120,54 @@ class MFunctor t where
     -}
     hoist :: (Monad m) => (forall a . m a -> n a) -> t m b -> t n b
 
+{-| Convert a Monad to an isomorphic transformation of Identity
+> mtrans (a >> b)   = mtrans a >> mtrans b
```

Comment:
It might be easier to say that `mtrans` is a monad morphism since the definition of monad morphism is already present within this same module.  That subsumes this statement and generalizes it (because it would also imply that `mtrans (return x) = return x`)

Reply:
I don't really know. I'm too new at this.

Reply:
Yeah, I'm reasonably certain that all of these instances qualify as monad morphisms.  It's also consistent with everything else in the library (thus the name: `mmorph` = "monad morphism")